### ISSUE TITLE
Hong Kong flag missing

### ISSUE THREAD

**Issue by flinbu** [kud/flags-of-the-world#2]
Hong Kong is, in some legal terms, a country so it has is own ISO code that is `HK`

Can it be included in the repo?

**Comment by kud** [comment]
Yeah sure, we need to find the flag in .svg and we'll include it.

PR are welcome. 😊

**Comment by kud** [comment]
Closing — this project is archived and no longer actively maintained. Thanks for taking the time to raise this.
